buildscript/build.go

```diff
@@ -51,18 +51,12 @@ func TestKerberos(ctx *task.Context) error {
 
 // TestIntegration is an Executor that runs all integration tests for the provided packages.
 func TestIntegration(ctx *task.Context) error {
-	// TODO: Replace func body with following line in TOOLS-2807.
-	// return runTests(ctx, selectedPkgs(ctx), testtype.IntegrationTestType)
-
-	// Don't run integration tests in /common yet.
-	pkgs := selectedPkgs(ctx)
-	for i := range pkgs {
-		if pkgs[i] == "common" {
-			pkgs = append(pkgs[:i], pkgs[i+1:]...)
-			break
-		}
+	err := runTests(ctx, selectedPkgs(ctx), testtype.IntegrationTestType)
+	if err != nil {
+		return err
 	}
-	return runTests(ctx, pkgs, testtype.IntegrationTestType)
+
+	return runTests(ctx, []string{"common"}, testtype.SRVConnectionStringTestType)
```

Comment:
I'm not sure these should be grouped in with the rest of the integration tests. I think these are similar to the Kerberos or AWS tests, in that they have different MongoDB setup/connection requirements. As such, I think that we should probably run them in a separate target instead of bundling them with the integration tests

Reply:
Yea, I was thinking about this a bit. So this testtype only applies to one test (`TestOptionsParsingForSRV`) and this is the relevant setup code from mtc's Evergreen config:
```
if [ "${INT_TEST}" = "true" ]; then
    export TOOLS_TESTING_INTEGRATION="true"
    export TOOLS_TESTING_SRV="true"
    export ATLAS_URI=${atlas_srv_uri}
fi;
```
Based on that, the srv test should have identical req's as any integration test on Evergreen. And now that we always export `ATLAS_URI`, we could theoretically just change the testtype of `TestOptionsParsingForSRV` to an integration test.

Side note though -- this test only works locally if you're connected to the vpn (probably stemming from some dns lookup that the driver's connstring package does). That may be why there's two testtypes in the first place.

Still think it's worth it to separate it, or should we just make it an integration test? I'm okay either way.

Reply:
I think the local testing use-case is the main reason why we'd want to keep them as separate targets in `build.go` -- not only would you have to be on the VPN, but you'd also have to have the `ATLAS_URI` set, which I think is a secret expansion from the evg config.